### DISCUSSION TITLE
How can I tell if DRY is enabled?

### DISCUSSION THREAD

**Discussion by io7m** [LostRuins/koboldcpp#1058] [Q&A] [answered]
Hello!

I was struggling with models getting stuck in loops in `llamacpp` and noticed that there was an issue filed about implementing the DRY sampler. That lead me to `koboldcpp` which claimed to have already implemented it. I see an option in the UI for DRY, but there's no checkbox to turn it on, and the little tooltip icon suggests that it will be turned on "if supported".

How can I tell if DRY is being used, currently?

**Comment by LostRuins** [comment, marked as answer]
Set your DRY multiplier to a value greater than 0 to enable it

**Reply by io7m** [reply]
Thanks!
